health_records/aks-engine/aks-engine-1.json

```diff
@@ -0,0 +1,549 @@
+[
```

Comment:
whare are these .json data files used ?

Reply:
I deleted these. they are used for e2e testing on my local machine